mod-invoice-storage/schemas/batch_voucher.xsd

```diff
@@ -0,0 +1,237 @@
+<?xml version="1.0" encoding="utf-8"?>
+<xs:schema attributeFormDefault="unqualified"
+           elementFormDefault="qualified"
+           xmlns:xs="http://www.w3.org/2001/XMLSchema">
+
+  <xs:element name="batchVoucher" type="BatchVoucherType">
+    <xs:annotation>
+      <xs:documentation>Batch voucher</xs:documentation>
+    </xs:annotation>
+  </xs:element>
+
+
+  <xs:complexType name="BatchVoucherType">
+    <xs:annotation>
+      <xs:documentation>Top-level batch voucher, i.e. a collection of batched vouchers</xs:documentation>
+    </xs:annotation>
+    <xs:sequence>
+      <xs:element name="id" type="UUIDType" minOccurs="0">
+        <xs:annotation>
+          <xs:documentation>UUID of batch voucher</xs:documentation>
+        </xs:annotation>
+      </xs:element>
+      <xs:element name="batchGroup" type="xs:string">
+        <xs:annotation>
+          <xs:documentation>The human-readable name of the batch group used when generating this batch voucher</xs:documentation>
+        </xs:annotation>
+      </xs:element>
+      <xs:element name="created" type="xs:dateTime">
+        <xs:annotation>
+          <xs:documentation>When this batch voucher was created</xs:documentation>
+        </xs:annotation>
+      </xs:element>
+      <xs:element name="start" type="xs:dateTime">
+        <xs:annotation>
+          <xs:documentation>This export is for all vouchers between start and end</xs:documentation>
+        </xs:annotation>
+      </xs:element>
+      <xs:element name="end" type="xs:dateTime">
+        <xs:annotation>
+          <xs:documentation>This export is for all vouchers between start and end</xs:documentation>
+        </xs:annotation>
+      </xs:element>
+      <xs:element name="batchedVouchers">
+        <xs:annotation>
+          <xs:documentation>A list of batch voucher records</xs:documentation>
+        </xs:annotation>
+        <xs:complexType>
+          <xs:complexContent>
+            <xs:extension base="BatchedVouchersType"/>
+          </xs:complexContent>
+        </xs:complexType>
+      </xs:element>
+      <xs:element name="totalRecords" type="xs:unsignedByte">
```

Comment:
Maximum value for `type="xs:unsignedByte"` is 255. 
@craigmcnally, could you confirm it will be enough for future usage?

Reply:
> Maximum value for `type="xs:unsignedByte"` is 255.
> @craigmcnally, could you confirm it will be enough for future usage?

Thanks. I am not sure, but the simplest way to change a type